salt/loader.py

```diff
@@ -1337,14 +1355,45 @@ def _load_module(self, name):
                         self.tag,
                         name)
                 if suffix == '':
-                    mod = imp.load_module(mod_namespace, None, fpath, desc)
+                    if six.PY3:
```

Comment:
Good job @skizunov , I should send this to Eric Snow, the author of the importlib stuff, so he can see why I used imp in the first place!
Looks like you have covered all the cases